test/test_container.py

```diff
@@ -0,0 +1,27 @@
+import os
+import subprocess
+
+import pytest
+
+from containerbuild import build_container_fixture
+
+
+@pytest.mark.skipif(os.getuid() != 0, reason="needs root")
+def test_container_builds_image(tmp_path, build_container):
+    output_dir = tmp_path / "output"
+    output_dir.mkdir()
+    subprocess.check_call([
+        "podman", "run",
+        "--privileged",
+        "-v", f"{output_dir}:/output",
+        build_container,
+        "build",
+        "minimal-raw",
+        "--distro", "fedora-41"
```

Comment:
Can we do something quick and dirty to pave over the RPM downloading issue that our stack is suffering from? Override repos with rpmrepo.osbuild.org for the test?

Reply:
I will try centos-9 instead, from my experience with bib it seems the centos mirrors are generally a bit more stable.